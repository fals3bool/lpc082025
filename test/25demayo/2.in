5
Batata 0.40
Cebolla 0.30
Oregano 0.01
Porotos 0.50
Tomate 0.30
5 4
Batata 1.20
Cebolla 1.20
Oregano 0.04
Porotos 1.50
Tomate 1.20
