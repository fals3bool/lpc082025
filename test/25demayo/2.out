Batata 0.4
Porotos 0.5
